src/style.css

```
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  align-items: stretch;
+}
+
+.block {
+  height: 300px;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-grow: 1;
+}
+
+.block:nth-child(1) {
+  background-color: rgb(255, 0, 0);
+}
+
+.block:nth-child(2) {
+  background-color: rgb(204, 0, 0);
+}
+
+.block:nth-child(3) {
+  background-color: rgb(153, 0, 0);
+}
+
+.block:nth-child(4) {
+  background-color: rgb(102, 0, 0);
+}
+
+.block:nth-child(5) {
+  background-color: rgb(51, 0, 0);
+}
+
+.block:nth-child(6) {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 300px) {
+  .container .block {
+    order: 0;
+    width: 100%;
+  }
+}
+@media (min-width: 600px) {
+  .block {
+    width: 50%;
+    flex-basis: 50%;
+  }
+  .block:nth-child(4) {
+    order: 3;
+  }
+  .block:nth-child(3) {
+    order: 4;
+  }
+  .block:nth-child(5) {
+    order: 5;
+  }
+  .block:nth-child(6) {
+    order: 6;
+  }
+}
+@media (min-width: 900px) {
+  .block {
+    width: 33%;
+    flex-basis: 33%;
```

```suggestion
    flex-basis: 33%;
```
Коли використовуєш flex-basis, то width вже не потрібна